.circleci/config.yml

```diff
@@ -57,6 +57,7 @@ jobs:
     steps:
       - setup-node:
           node-version: <<parameters.node-version>>
+      - run: npm run compile
```

Comment:
Is this needed? I think `test:ci` should trigger a `pretest` that should be good enough?

Or I guess the answer is yes because of the precompile/postcompile steps... but I wonder if we could hook that into the same thing pretest does by sharing some scripts appropriately. btw precompile.mjs refers to `postinstall`.